package.xml

```diff
@@ -17,13 +17,10 @@
 
   <build_depend>console_bridge_vendor</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
-  <build_depend>libpoco-dev</build_depend>
-  <build_depend>poco_vendor</build_depend>
+  <depend>rcpputils</depend>
```

Comment:
NItpick: please move this line in the order according to the schema.

Reply:
Fixed https://github.com/ros/class_loader/pull/139/commits/139b38a1ec340e76b63e4d2748306edbdc798241